README.md

```diff
@@ -325,6 +326,191 @@ const commonProps = {
 
 A warning is displayed when no fallback image is passed. This warning can be disabled in special circumstances. To disable this warning, look in the [warnings section](#warnings).
 
+#### ImgixProvider Component
+
+> New 🪄 ✨
```

Comment:
Not sure if this line was intentional but I think we should steer away from labeling features as "new" or otherwise because they will remained tied to a certain point in time.